tst/EnergyPlus/unit/HybridModel.unit.cc

```diff
@@ -0,0 +1,351 @@
+// You are under no obligation whatsoever to provide any bug fixes, patches, or upgrades to the
+// features, functionality or performance of the source code ("Enhancements") to anyone; however,
+// if you choose to make your Enhancements available either publicly, or directly to Lawrence
+// Berkeley National Laboratory, without imposing a separate written license agreement for such
+// Enhancements, then you hereby grant the following license: a non-exclusive, royalty-free
+// perpetual license to install, use, modify, prepare derivative works, incorporate into other
+// computer software, distribute, and sublicense such enhancements or derivative works thereof,
+// in binary and source code form.
+
+// EnergyPlus::ZoneTempPredictorCorrector Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+#include "Fixtures/EnergyPlusFixture.hh"
+
+// EnergyPlus Headers
+#include <EnergyPlus/DataHeatBalance.hh>
+#include <EnergyPlus/DataHeatBalFanSys.hh>
+#include <EnergyPlus/DataGlobals.hh>
+#include <EnergyPlus/DataZoneControls.hh>
+#include <EnergyPlus/DataZoneEquipment.hh>
+#include <EnergyPlus/DataZoneEnergyDemands.hh>
+#include <EnergyPlus/DataSizing.hh>
+#include <EnergyPlus/HeatBalanceManager.hh>
+#include <EnergyPlus/UtilityRoutines.hh>
+#include <EnergyPlus/ZonePlenum.hh>
+#include <EnergyPlus/ZoneTempPredictorCorrector.hh>
+#include <EnergyPlus/DataLoopNode.hh>
+#include <EnergyPlus/DataHVACGlobals.hh>
+#include <EnergyPlus/DataSurfaces.hh>
+#include <EnergyPlus/DataEnvironment.hh>
+#include <EnergyPlus/DataAirflowNetwork.hh>
+#include <EnergyPlus/Psychrometrics.hh>
+#include <EnergyPlus/ScheduleManager.hh>
+#include <EnergyPlus/DataRoomAirModel.hh>
+#include <EnergyPlus/HybridModel.hh>
+#include <EnergyPlus/DataPrecisionGlobals.hh>
+
+using namespace EnergyPlus;
+using namespace ObjexxFCL;
+using namespace EnergyPlus::DataHeatBalance;
+using namespace EnergyPlus::DataHeatBalFanSys;
+using namespace DataGlobals;
+using namespace EnergyPlus::DataZoneControls;
+using namespace EnergyPlus::DataZoneEquipment;
+using namespace EnergyPlus::DataZoneEnergyDemands;
+using namespace EnergyPlus::DataSizing;
+using namespace EnergyPlus::HeatBalanceManager;
+using namespace EnergyPlus::ZonePlenum;
+using namespace EnergyPlus::ZoneTempPredictorCorrector;
+using namespace EnergyPlus::DataLoopNode;
+using namespace EnergyPlus::DataHVACGlobals;
+using namespace EnergyPlus::DataSurfaces;
+using namespace EnergyPlus::DataEnvironment;
+using namespace EnergyPlus::DataAirflowNetwork;
+using namespace EnergyPlus::Psychrometrics;
+using namespace EnergyPlus::ScheduleManager;
+using namespace EnergyPlus::DataRoomAirModel;
+using namespace EnergyPlus::HybridModel;
+using namespace EnergyPlus::DataPrecisionGlobals;
+
+TEST_F( EnergyPlusFixture, ZoneTempPredictorCorrector_CorrectZoneAirTempTest )
+{
+
+	// ZoneTempPredictorCorrector variable initialization 
+	std::vector< int > controlledZoneEquipConfigNums;
+	controlledZoneEquipConfigNums.push_back( 1 );
```

Comment:
@sanghlee-lbnl I think I see the problem.  These two lines are not needed here, because `controlledZoneEquipConfigNums` is declared locally in [CorrectZoneAirTemp](https://github.com/NREL/EnergyPlus/blob/dd0a2e6162b5b5ee17fccfcd20c5bef95fba1df3/src/EnergyPlus/ZoneTempPredictorCorrector.cc#L3890-L3894)
Instead, I think you need `Zone( 1 ).IsControlled = true; `  down where you're initializing the other Zone variables.